### ISSUE TITLE
MFDs/Displays will not turn on

### ISSUE THREAD

**Issue by AlpacAttac** [flybywiresim/a32nx#3941]
**Mod Version**
 v0.5.3

**Describe the bug**
 
All screens/MFDs or other displays are black.

**To Reproduce**

1. Start the latest version of MSFS2020
2. Load into the Airbus A320neo
3. After completing ground power the screens are still dark

**Expected behavior**

The MFDs/Displays to turn on.

**Actual behavior**

The MFDs/Display did not turn on and remained black.

**References**

![image](https://user-images.githubusercontent.com/65864537/110948080-eafa8700-830e-11eb-8e4d-142270f29696.png)

**Additional context**

Was this working before/when did the issue start occurring?
This was working before, I only stated having this problem after the newest flight sim update
Is this a problem in the vanilla unmodded game?
There are no other mods beside A32NX installed

Discord username (if different from GitHub): AlpacAttac#0849

**Comment by MisterChocker** [comment]
Duplicate of #3938

**Comment by n3rdcoach** [comment]
same problem here

**Comment by MisterChocker** [comment]
Using this issue report since op filled our the template.

**Comment by MisterChocker** [comment]
Please use the development build until we update the stable version.

**Comment by Jimothy1st** [comment]
Why was my post closed when this was a duplicate of my problem!? Post #3938
If I did something wrong please explain, I’m new and that was my first post
Switching to the development version or the custom version does not resolve my issue, all displays are still blank.  I have uninstalled the mod and re-installed and still no displays.

**Comment by Benjozork** [comment]
The development version works correctly. Your installer might be installing the wrong thing, I've seen it happen.

Can you provide a wide screenshot of your cockpit ?

**Comment by Jimothy1st** [comment]
There is a screenshot in my original post #3938
You actually commented on that post earlier today before it was closed.
Interesting that you think the installer isn’t working correctly, not sure if it’s relevant but the game and content are on my D:\ could that cause problems with the installer?
I will however download the development version zip file and install manually, fingers crossed that will resolve the problem.

**Comment by Jimothy1st** [comment]
> The development version works correctly. Your installer might be installing the wrong thing, I've seen it happen.
> 
> Can you provide a wide screenshot of your cockpit ?

That hasn't worked, problem still persists
![Development Version](https://user-images.githubusercontent.com/80516927/110977800-7f3e0b80-835a-11eb-8c3a-270b311c9d56.jpg)

**Comment by Benjozork** [comment]
You are somehow on the stable version.

Can you try a manual install using one of the direct links on our website ?

**Comment by Benjozork** [comment]
Are you sure you are unzipping into the right `Community/` folder ?

I have seen people mix them up because they reinstalled MSFS at some point and picked a different Community folder, with the old one remaining.

I can tell you are on stable with the old SELCAL font:

**Comment by Michael170978** [comment]
I had the same issue and I could  help myself. What I did. I removed all mods from my community folder . Then installed the develop version of A320. Started the game and all dispalys worked fine. Afterthat I removed back all other Mods exceptly Mod A321 & A330 to the community folder.

**Comment by Myjaguar27** [comment]
Hi - I have encountered the same problem and am still learning the flight sim world. My earlier install worked with the Stable version but, after latest msfs update, I lost all instrument displays and, after uploading a simbrief flight plan and updating arrival/departure information on mcdu the "flight plan" locks up and can't be selected. I've installed the development version and the instrumentation is now visible but the "flight plan" still locks up after entering arrival info. Also , when selecting "AP" the "Auto Throttle Control" has to be selected when previously it was connected when AP selected. I purchased via Steam if that's relevant and locate my addons using %appdata% - MSFS  Any help appreciated.

**Comment by Funkythrusta** [comment]
Hi there, love the project and congrats to the developer community! Got 100+ flight hrs in the log with A32nx mod. After recent MS2020 update to 1.14.5.0 all displays remain black with all power units Bat, APU on. Confirming behaviour described in this thread. Any advice appreciated!

**Comment by andrejetcat** [comment]
I can confirm this issue, BUT today i test it 3 times. First time, it works good, all displays are on. secound time, without ANY changes on the system or in the COM-folder, both ND´s remains off. Third time, also without any changes, only the ND on the F/O´s side remains black. 
strange and hard to reproduce :(

**Comment by Benjozork** [comment]
> Hi there, love the project and congrats to the developer community! Got 100+ flight hrs in the log with A32nx mod. After recent MS2020 update to 1.14.5.0 all displays remain black with all power units Bat, APU on. Confirming behaviour described in this thread. Any advice appreciated!

Have you updated to the development version of the A32NX?

**Comment by Benjozork** [comment]
> I can confirm this issue, BUT today i test it 3 times. First time, it works good, all displays are on. secound time, without ANY changes on the system or in the COM-folder, both ND´s remains off. Third time, also without any changes, only the ND on the F/O´s side remains black. 
> strange and hard to reproduce :( 

I can reproduce random avionics crashes (shown by random Thales Avionics maintenance screens or blank displays).

Seem to be caused by race conditions in dependency loading. `import-async="false"` seems to not work anymore.

**Comment by Funkythrusta** [comment]
> > Hi there, love the project and congrats to the developer community! Got 100+ flight hrs in the log with A32nx mod. After recent MS2020 update to 1.14.5.0 all displays remain black with all power units Bat, APU on. Confirming behaviour described in this thread. Any advice appreciated!
> 
> Have you updated to the development version of the A32NX?

Thanks for confirming the bug and the hint. Usually I fly the stable build. Tried the developper build as suggested. Same problem. Interestingly the EFB Display is at least flickering white, so not dark. Love the coffee cup though. I re-intalled MS2020 to exclude any other root cause. A320nx ist the only package in my community folder. All cockpit displays remain dark!

**Comment by Adi-18** [comment]
Same to me. Bevore Update MSFS to 1.14.5.0, it works well.
It occurs in dark-and-cold **and** also start to flight directly on runway.
I see, there is no battery voltage displaying after activate external power.

Original A320neo works, but new the battery voltage shows only after activate external power in cold-and-dark mode.

**Comment by kl0r0lle** [comment]
doesnt work with FS v1.14.5.
will have to wait for update. 

Another mistake is the excessive air resistance or engine power. It is no longer possible to rise above fl320

**Comment by Benjozork** [comment]
People, the development version works. It's been confirmed and if it doesn't work for you then you have a separate problem.

Please use our Discord support channel as GitHub is not appropriate in this case for user support.

**Comment by andrejetcat** [comment]
![ND_A32NX](https://user-images.githubusercontent.com/72918017/111083804-907c4900-850f-11eb-8040-018fc59e3db2.jpg)
 downloaded just few secounds ago. DEV version over this side, not installer.

**Comment by jasonschock** [comment]
Using the installer on dev (cd44024) fixed this for me.

**Comment by Adi-18** [comment]
Jep, thank you, this works for me. But now I have new problem: I cannot select and enter NAV → ILS Also not for NAV → VOR.
I did it for VFR.
![image](https://user-images.githubusercontent.com/29123730/111147980-623c4f00-858b-11eb-8e95-2ac75b167c74.png)

**Comment by andrejetcat** [comment]
Problem may connected with some liverys. I testet it today with the LATEST DEVELOPMENT version with a custom livery. Both ND´s still remains black. 
Tested again with FBW-Livery, both ND´s are back.

**Comment by Viopassch** [comment]
2 day ago my livery worked without the issue above.
With the actual dev version not.

**Comment by Benjozork** [comment]
We do not guarantee compatibility with any liveries and it's the responsibility of the authors to make sure they work.

The stable version is available for users who don't want unexpected compatibility breakage. I don't know what you expect from a *development* version.

**Comment by tracernz** [comment]
> 
> 
> Because of "problem with MFDs/Displays will not turn on" on stable version? 😉

There is a new stable release (0.5.4) with these issues fixed. ;)

**Comment by Pym15** [comment]
Je rencontre le même problème que plusieurs personne ! AlpacAttac ; tracernz ; Adi-18 ;  MisterChocker

Version du mod v0.5.3
Décrivez le bogue

Tous les écrans / MFD ou autres affichages sont noirs.

Reproduire

Démarrez la dernière version de MSFS2020
Charger dans l'Airbus A320neo
Après avoir terminé l'alimentation au sol, les écrans sont toujours sombres
Comportement prévisible

Les MFD / Écrans à allumer.

Comportement réel

Les MFD / Affichage ne se sont pas allumés et sont restés noirs.

**Comment by Benjozork** [comment]
@Pym15 Vous devez utiliser soit la version 0.5.4 ou la version développeur si vous rencontrez ce problème.

**Comment by Pym15** [comment]
Merci
je viens de télécharger la dernière version (FBW installer) et je rencontre toujours le problème

**Comment by Pym15** [comment]
@Benjozork 
merci de votre réponse,  pouvez vous m'indiquer un lien ou je peux trouver la version 0.5.4 car la dernière version stable que je trouve est apparemment la 0.5.2 !
cordialement.

**Comment by Benjozork** [comment]
Bonjour, si vous allez sur https://flybywiresim.com/ et descendez vers "Direct Download" vous trouverez le lien de téléchargement de la dernière 0.5.4 en format zip.

**Comment by Pym15** [comment]
merci beaucoup

**Comment by ashwad3** [comment]
Sometimes a little detail goes a long way. _It is not just liveries that you download that cause this issue_. It is the FlyByWire livery included in the package as well.

Correctly install the A32NX. Load the sim. Select the A320 **BUT** select the default livery. NOT the FlyByWire livery. The plane will load correctly with functioning MFDs.

FBW's livery is the issue.

**Comment by Benjozork** [comment]
FBW livery works perfectly fine for me. Make sure you have no other mods which can overwrite files in the A320neo.

**Comment by philw06** [comment]
same issue for me... Delete folder, reinstall through developer direct download. MFD still black
![Capture](https://user-images.githubusercontent.com/75388572/111867025-b1aac280-8971-11eb-8e3b-1cf23ac179df.JPG)

**Comment by Benjozork** [comment]
Try without any other things in your Community folder

**Comment by MisterChocker** [comment]
Closing since no response from the op.

**Comment by gabegeforce** [comment]
> Problem may connected with some liverys. I testet it today with the LATEST DEVELOPMENT version with a custom livery. Both ND´s still remains black. Tested again with FBW-Livery, both ND´s are back.

same here! I tried all the troubleshooting steps from the posts by flybywire website everything else works except the ND that either shows "If you are seeing this, instrument did not load" or just totally blank. the contrast knobs are all the way turned up, the stable neither the development works..do you have any other solution I can try? thanks in advance
